tests/unit/h/services/annotation_moderation_test.py

```diff
@@ -53,6 +56,8 @@ def test_set_status(self, svc, annotation, user):
             annotation.moderation_log[0].new_moderation_status
             == ModerationStatus.DENIED
         )
+        if with_slim:
+            assert annotation.slim.moderated == annotation.is_hidden
```

Comment:
This was happening before for unhide/hide via the annotation write service but not while setting the status directly. Fixed here.